m4_sessions/1706-inning/group_three.md

```diff
@@ -1,9 +1,48 @@
 # Group Three
 
-### Session Title:
+### Session Title: Role of tech in fake news (working title)
 
 ### Session Outcomes: 
++ Be aware of what tech organizations are doing (facebook, news organizations)
++ Identify motives behind fake news, profit vs. political sway
++ Be aware of traditional news fact checking and processes
+
 
 ### Session Outline:
+10 min: intro, name game, explain session outcomes - Colleen
+10 min: outside materials - Adam
++ Factitious quiz
++ Show fake Obama lip sync
++ 60 min. video about twitter bot if time.
+20 min: group discussion/ split into pro/con groups - Jack explain. participate in discussions
++ Pro censorship vs. anti intervention group
++ Each group has to come up with three best arguments 
+10 min: back together conclusions/ debate - All contribute
+10 min: wiggle room/ reflections - 
++ Reflection question, what would you do if you were at a company that was involved in fake news
++ What obligation do you think that tech companies have in mderating fake news.
+
 
 ### Names of group members:
+@jwpincus
+@colleenw
+@seth_a_t
+@adam_carpenter
+@zaner
+
+
+### Resources:
```

Comment:
Maybe send these out to your group on Monday/Tuesday because these articles are super interesting.  I think this will also help people form opinions in the debate.